src/backend/efiling-reviewer-api/src/main/java/ca/bc/gov/open/jag/efilingreviewerapi/Keys.java

```diff
@@ -15,6 +15,9 @@ private Keys() {}
     public static final Map<String, String> ACCEPTED_DOCUMENT_TYPES = new HashMap<String, String>() {{
         put("RCC", "Response to Civil Claim");
     }};
+    public static final Map<String, String> RESTRICTED_DOCUMENT_TYPES = new HashMap<String, String>() {{
+        put("TEST", "This is a temporary");
```

Comment:
This is a temporary test?

Reply:
I don't have any of the restricted document types. So for now I put a value in.